cpp/src/io/json/json_tree.cu

```diff
@@ -254,12 +308,423 @@ tree_meta_t get_tree_representation(device_span<PdaTokenT const> tokens,
                                              is_node);
   CUDF_EXPECTS(parent_node_ids_end - parent_node_ids.begin() == num_nodes,
                "parent node id gather mismatch");
+  nvtxRangePop();
+  nvtxRangePop();  // node_parent_ids
   return {std::move(node_categories),
           std::move(parent_node_ids),
           std::move(node_levels),
           std::move(node_range_begin),
           std::move(node_range_end)};
 }
 
+std::tuple<rmm::device_uvector<NodeIndexT>, rmm::device_uvector<size_type>>
+records_orient_tree_traversal(device_span<SymbolT const> d_input,
```

Comment:
Can this function be broken up? It is over 400 lines. It is full of debugging info that doesn't seem like it should be merged.

Reply:
Yes. will break it up. There are many reorders happening. Each function should be aware of pre-condition. will break it up.

Reply:
done.